templates/tfengine/components/cicd/triggers.tf

```diff
@@ -1,327 +0,0 @@
-# Copyright 2021 Google LLC
```

Comment:
I think this one made it into the repo again after some merge I did. I'm removing it again but It wasn't supposed to be here in the first place